config/config.go

```diff
@@ -135,7 +146,7 @@ func (c *Config) Read(file string) error {
 		return err
 	}
 
-	f, err := os.Open(file)
+	f, err := os.Open(c.File)
```

Comment:
I completely agree. I was thinking about making this change a couple of days ago, but then got mired into a big refactoring that I ended up throwing away.